iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergSnapshotOperations.java

```diff
@@ -116,4 +117,47 @@ public void testReplaceBranchWithSnapshot() {
     result = shell.executeStatement("SELECT COUNT(*) FROM default.testReplaceBranchWithSnapshot.branch_branch1");
     assertEquals(6L, result.get(0)[0]);
   }
+
+  @Test
+  public void testReplaceTag() {
```

Comment:
We already have `TestHiveIcebergTagOperation `& `TestHiveIcebergBranchOperation`. Should we put the branch&tag operation into there?